bluetooth/src/main/java/me/aflak/bluetooth/Bluetooth.java

```diff
@@ -315,11 +336,33 @@ public boolean isConnected(){
     }
 
     /**
-     * Send string message to the connected device.
+     * Send string message to all the connected devices.
      * @param msg String message.
      * @param charset Charset used to encode the String. Default charset is UTF-8.
      */
     public void send(String msg, String charset){
```

Comment:
Same here, you shouldn't send the message to every socket. Or rename it `sendToAll()`